cli/flox-rust-sdk/src/providers/build.rs

```diff
@@ -2300,6 +2300,38 @@ mod tests {
         let (flox, tmpdir) = flox_instance();
         assert_manifest_build_succeeds(GENERATED_DATA.join("build/hello"), "hello", &flox, tmpdir);
     }
+
+    /// Test that a runtime package installed to an "other" system type does
+    /// not trigger a build failure (#3055).
+    #[test]
+    fn other_system_runtime_packages() {
```

Comment:
(non-blocking) Grouping this with the other `runtime-packages` tests further up the file would make the tests easier to navigate.